python/humbug/report.py

```diff
@@ -332,9 +339,12 @@ def packages_report(
             tags = []
         tags.append("type:dependencies")
 
-        available_packages = [
-            str(package_info) for package_info in pkg_resources.working_set
-        ]
+        available_packages = []
```

Comment:
@Andrei-Dolgolev : Can you put a message in the report content saying that `pkg_resources` was not available?

Also maybe a tag saying that `packages_report` failed.